ec/ecresource/deploymentresource/elasticsearch/v2/schema.go

```diff
@@ -150,6 +150,32 @@ func ElasticsearchSchema() schema.Attribute {
 				Optional:    true,
 				Validators:  []validator.String{stringvalidator.OneOf(strategiesList...)},
 			},
+
+			"keystore_contents": keystoreContentsSchema(),
+		},
+	}
+}
+
+func keystoreContentsSchema() schema.Attribute {
```

Comment:
So, we're only planning to manage keystore items which are added to the deployment. That is, we won't remove keystore items added outside of the deployment resource?

Just making sure I'm reading the rest of the code correctly. I agree the above is the right approach.

Reply:
Basically just understanding the lack of `computed: true` here as well.

Reply:
Yeah, the `ec_deployment` resource should handle only keystore items that are declared in its attribute and shouldn't interfere with the items managed  by `ec_deployment_elasticsearch_keystore` resources. The new IT test should cover this use case.
Also the keystore API is write only so the provider cannot read secrets.